pkg/kp/rcstore/store.go

```diff
@@ -33,7 +33,10 @@ type Store interface {
 	// session that deletes its keys on invalidation.
 	Lock(id fields.ID, session string) (bool, error)
 
+	// Set the desired replica count for the given RC to the given integer.
 	SetDesiredReplicas(fields.ID, int) error
```

Comment:
yes I think both are useful. Add happened to be an efficiency win for the code I'm writing at the moment, but I think Set is a more obvious interface for humans (rctl), and it's also the only way to clear an RC to zero in a single call.